skills_DS/skills_DS/settings.py

```diff
@@ -87,7 +87,7 @@
         'NAME': 'skills_analysis',
         'USER': 'postgres',
         'PASSWORD': 'example',
-        'HOST': 'postgres',
+        'HOST': '127.0.0.1',
```

Comment:
The host needs to be `postgres` for it to work in Docker. Let me know if you need help setting up the Docker enviorment.